containers/themes/ThemeCard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
I don't see the value of creating this component instead of directly using `RadioCard`.

Reply:
Yes, originally `ThemeCard` was a bit different, that's why it was a different component. I added a `customizable` prop now, so I think it makes sense to have the component again.